cart/src/components/Home.js

```diff
@@ -0,0 +1,18 @@
+import CartData from "./Data";
+import "../styles/Home.css";
+
+const Home = ({ addItemToCart }) => {
+  const cards = CartData.map((items) => (
```

Comment:
don't use a variable to store/return JSX expressions, directly use this under the <div>

`return <div className="cards">{CartData.map((item) => (.....)}</div>;`